README.md

```diff
@@ -1,12 +1,26 @@
 # PS4_db_rebuilder
-ps4 built in database rebuilder has the tendency to remove fpkg from the database, this will repopulate the database with them
 
-## requirement
+Ps4 built-in database rebuilder has the tendency to remove fpkg from the database. This will repopulate the database with them!
+
+## Requirement
+
 - Python (preferably 3.0+)
 
-## how-to-use
-1) recurively clone this repo or download a [Release](https://github.com/Zer0xFF/PS4_db_rebuilder/releases)
-2) start FTP server on the PS4
-3) run the python script though terminal/cmdline `python3 fix_db.py PS4_IP`
-4) wait for the script to finish, then logout of the PS4 user without closing the browser
-5) log back in and all your games should be there again
+
+## Instructions
+
+**NOTE**: Sony seems to have modified the PS4 DB from 5.05 to the 6.72 system version. Taking this into account, follow the 3) step accordingly to your system. Versions after 5.05 may have to follow the step accordingly to the 6.72 system version.
+
+1) Clone this repo or download a [Release](https://github.com/Zer0xFF/PS4_db_rebuilder/releases)
```

Comment:
just cloning the repo wont work, it has to be recurive clone

Reply:
Right, will make the changes!